### DISCUSSION TITLE
Welcome to UnityWebBrowser Discussions!

### DISCUSSION THREAD

**Discussion by Voltstro** [Voltstro-Studios/UnityWebBrowser#32] [Announcements]
## Hello, and welcome to UWB discussions!
Hey everyone!

We have decided to enable discussions on this project as we have been getting some repeated questions about this project in private messages. Asking your questions here means that others in the future with the same or a similar question can see it, as well as any answers for it (and maybe even contribute towards the question).

- So if you have any issues, please still use the [GitHub issue tracker](https://github.com/Voltstro-Studios/UnityWebBrowser/issues) for this project, but for general questions use this.
- If you have an ideas guy moment, you can also [start a discussion](https://github.com/Voltstro-Studios/UnityWebBrowser/discussions/categories/ideas) for it here, to have others discuss about it with you, and even possibly get it added to UWB.

Anyway, thanks for having interest in this project.
-- From the Voltstro-Studios team.

**Comment by shubhamnegi1234** [comment]
can please someone teach me how to use it ?
i want to apply this on a cube surface on my project 
kindly help me or provide some tutorial or something

**Reply by Voltstro** [reply]
> can please someone teach me how to use it ?

Using UWB as a whole? [See the docs](https://projects.voltstro.dev/UnityWebBrowser/).

> i want to apply this on a cube surface on my project

You would need to dynamically apply the generated [BrowserTexture](https://projects.voltstro.dev/UnityWebBrowser/api/voltstrostudios.unitywebbrowser.core/voltstrostudios.unitywebbrowser.core.webbrowserclient.browsertexture/#voltstrostudios-unitywebbrowser-core-webbrowserclient-browsertexture) (which is a Texture2D) to a material. The texture gets update automatically by the client. Material should update automatically once you have set it. You would probably also want to create a custom [BaseUwbClientManager](https://projects.voltstro.dev/UnityWebBrowser/api/voltstrostudios.unitywebbrowser.core/voltstrostudios.unitywebbrowser.core.baseuwbclientmanager/) implementation, so you don't need a RawTexture component (as both default implementations do require a RawTexture).

Also side note, you really should create a new discussion for something like this, instead of just replying to an announcement post.
